Source/WebCore/css/StyleProperties.cpp

```diff
@@ -486,13 +431,13 @@ static std::optional<CSSValueID> fontStretchKeyword(double value)
     return fontStretchKeyword(valueAsFontSelectionValue);
 }
 
-String StyleProperties::fontValue() const
+String StyleProperties::fontValue(const StylePropertyShorthand& shorthand) const
```

Comment:
Could we assert that we're always passing in `fontShorthand()`?

Reply:
Like `ASSERT(shorthand.id() == CSSPropertyFont)`?

Reply:
Yeah that works.